packages/web-components/src/text/text.stories.ts

```diff
@@ -118,3 +118,133 @@ export default {
 } as TextStoryMeta;
 
 export const Text = renderComponent(storyTemplate).bind({});
+
+//
+// Attribute stories
+//
+
+export const Nowrap = renderComponent(html<TextStoryArgs>`
+  <fluent-text nowrap>
+    <div style="display: block; width: 320px; border: 1px solid black;">
+      This text will not wrap lines when it overflows the container.
+    </div>
+  </fluent-text>
+`);
+
+export const Truncate = renderComponent(html<TextStoryArgs>`
+  <fluent-text truncate nowrap>
+    <div style="display: block; width: 320px; border: 1px solid black;">
+      Setting <code>truncate</code> and <code>nowrap</code> will truncate when it overflows the container.
+    </div>
+  </fluent-text>
+`);
+
+export const Italic = renderComponent(html<TextStoryArgs>`
+  <fluent-text italic>
+    <span>Italics are emphasized text.</span>
+  </fluent-text>
+`);
+
+export const Underline = renderComponent(html<TextStoryArgs>`
+  <fluent-text underline>
+    <span>Underlined text draws the reader's attention to the words.</span>
+  </fluent-text>
+`);
+
+export const Strikethrough = renderComponent(html<TextStoryArgs>`
+  <fluent-text strikethrough>
+    <span>Strikethrough text is used to indicate something that is no longer relevant.</span>
+  </fluent-text>
+`);
+
+export const Block = renderComponent(html<TextStoryArgs>`
+  <span>
+    <fluent-text style="background: #ddd"><span>Fluent text is inline by default. Setting</span></fluent-text>
+    <fluent-text style="background: #ddd" block><span>block</span></fluent-text>
+    <fluent-text style="background: #ddd"><span>will make it behave as a block element.</span></fluent-text>
+  </span>
+`);
+
+// TODO: `as` doesn't appear to be attribute or feature but it is being documented as such. If this is true, it should be moved/removed.
```

Comment:
Yep, this was simply a method to illustrate setting the actual text element value. We should remove in favor of the below.